public/css/quiz-rule.css

```diff
@@ -0,0 +1,81 @@
+@import url('https://framer-ui.netlify.app/components/framer.css');
+
+:root{
+    --navbar-bg:#2C2929;
+    --img-text:#f1f2f3;
+    --img-banner:#4F46E5;
+
+}
+
+.navbar-bg{
+    background: var(--navbar-bg);
+}
+
+.nav-zindex{
+    z-index: 9;
+}
+.img-category{
+   width: 100%;
+   height: 15.625rem;
+   border-radius: 1.375rem;
+   object-fit: cover;
+}
+.img-text-left{
+    position: absolute;
+    top: 1.2rem;
+    right: -0.2rem;
+    color: var(--img-text);
+    font-weight: 500;
+    font-size: 1.2rem;
+    padding: 5px 0.75rem;
+    background: var(--img-banner);
```

Comment:
can use px values

Reply:
> can use px values

@manojsarna as Vikram told in his session if it is more than 10px then use rem